app/models/authenticator/httpd.rb

```diff
@@ -146,7 +150,7 @@ def user_details_from_headers(username, request)
         user_headers.each { |k, v| log_auth_debug("  %-24{key} = \"%{val}\"" % {:key => k, :val => v}) }
       end
 
-      groups     = CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[;:,]/)
+      groups     = CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[#{self.class.group_delimiters}]/)
```

Comment:
Actually, now that I wrote that we have to be resilient to users breaking stuff.  What if they set it to blank.

Also do we need to handle consecutive delimiters?  I had to handle all these cases in [floe's StringSplit](https://github.com/ManageIQ/floe/blob/master/lib/floe/workflow/intrinsic_function/transformer.rb#L235-L242), hence why I'm asking.

Reply:
I am not able to track down how to configure ldap group separators for Liberty.
The only delimiters I have been able to find in the Liberty xml configuration are single character.

Reply:
KeyCloak is our other reference system - can you set these values in KeyCloak?

Reply:
I see a bunch of double letter delimiters in keycloak.
Diving deeper

Reply:
ok. if the user sets to blank (the default value), then we will allow any of the 3 (the current behavior)